packages/react-docs/src/pages/components/modal.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import ComponentDocs from '../../components/componentDocs';
+import PropTypes from 'prop-types';
+import { Modal, Button } from '@patternfly/react-core';
+import Example from '../../components/example';
+
+class ModalDocs extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      showModal: false,
+      showLargeModal: false
+    };
+
+    this.hideModal = this.hideModal.bind(this);
```

Comment:
no need for this syntax.  Each of the below can be written as `hideModal = () => {` That will make them properties of the modal that are bound to the class instance already.